zh/deploy-on-gcp-gke.md

```diff
@@ -8,728 +8,346 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-gcp-gke/']
+### 新增节点组
 
-    例如：
+为 TiFlash/TiCDC 各自新增一个节点组。
 
-    {{< copyable "shell-regular" >}}
+{{< copyable "shell-regular" >}}
 
-    ```bash
-    gcloud compute instance-groups managed delete-instances gke-tidb-monitor-pool-08578e18-grp --instances=gke-tidb-monitor-pool-08578e18-c7vd --zone us-west1-b
-    ```
+```
+gcloud container node-pools create tiflash --cluster tidb --machine-type n1-standard-4 --num-nodes=1 \
+    --node-labels dedicated=tiflash --node-taints dedicated=tiflash:NoSchedule
+gcloud container node-pools create ticdc --cluster tidb --machine-type n1-standard-4 --num-nodes=1 \
+    --node-labels dedicated=ticdc --node-taints dedicated=ticdc:NoSchedule
+```
 
-## 销毁 TiDB 集群
+### 配置并部署
 
-如果你不想再继续使用 TiDB 集群，可以通过如下命令进行销毁：
+如果要部署 TiFlash，可以在 tidb-cluster.yaml 中配置 `spec.tiflash`，例如：
 
-{{< copyable "shell-regular" >}}
-
-```bash
-terraform destroy
+```yaml
+spec:
+  ...
+  tiflash:
+    baseImage: pingcap/tiflash
+    replicas: 1
+    storageClaims:
+    - resources:
+        requests:
+          storage: 100Gi
+    nodeSelector:
+      dedicated: tiflash
+    tolerations:
+    - effect: NoSchedule
+      key: dedicated
+      operator: Equal
+      value: tiflash
 ```
 
-> **注意：**
+> **警告：**
 >
-> 在执行 `terraform destroy` 过程中，可能会发生错误：`Error reading Container Cluster "tidb": Cluster "tidb" has status "RECONCILING" with message""`。当 GCP 升级 Kubernetes master 节点时会出现该问题。一旦问题出现，就无法删除集群，需要等待 GCP 升级结束，再次执行 `terraform destroy`。
+> 由于 TiDB Operator 会按照 `storageClaims` 列表中的配置**按顺序**自动挂载 PV，如果需要为 TiFlash 增加磁盘，请确保只在列表原有配置**末尾添加**，并且**不能**修改列表中原有配置的顺序。
 
-### 删除磁盘
+如果要部署 TiCDC，可以在 tidb-cluster.yaml 中配置 `spec.ticdc`，例如：
 
-如果你不再需要之前的数据，并且想要删除正在使用的磁盘，有以下两种方法可以完成此操作：
+```yaml
+spec:
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc
+    replicas: 1
+    nodeSelector:
+      dedicated: ticdc
+    tolerations:
+    - effect: NoSchedule
+      key: dedicated
+      operator: Equal
+      value: ticdc
+```
 
-- 手动删除：在 Google Cloud Console 中删除磁盘，或使用 `gcloud` 命令行工具执行删除操作。
+根据实际情况修改 `replicas` 等参数。
 
-- 自动删除：在执行 `terraform destroy` 之前将 Kubernetes 的 PV (Persistent Volume) 回收策略设置为 `Delete`，具体操作为在 `terraform destroy` 之前运行以下 `kubectl` 命令：
+最后使用 `kubectl -n tidb-cluster apply -f tidb-cluster.yaml` 更新 TiDB 集群配置。
 
-    {{< copyable "shell-regular" >}}
+更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
 
-    ```bash
-    kubectl --kubeconfig /path/to/kubeconfig/file get pvc -n namespace-of-tidb-cluster -o jsonpath='{.items[*].spec.volumeName}'|fmt -1 | xargs -I {} kubectl --kubeconfig /path/to/kubeconfig/file patch pv {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
-    ```
+## 使用本地存储
 
-    上述命令将获取 TiDB 集群命名空间中的 PVC (Persistent Volume Claim)，并将绑定的 PV 的回收策略设置为 `Delete`。在执行 `terraform destroy` 过程中删除 PVC 时，也会将磁盘删除。
+GCP 可以实例类型提供额外的 [本地存储卷](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ssd-instance-store.html)。可以为 TiKV 节点池选择这一类型的实例，以便提供更高的 IOPS 和低延迟。
```

Comment:
This is an **AWS link**. Is it correct?

Reply:
ah, thanks! I should link it to the GKE docs. Fixed.